packages/core/src/storage.ts

```diff
@@ -125,25 +129,47 @@ export function create({ token, host }: StorageConfig): StorageAPI {
         },
       });
       if (res.ok) {
-        const json = await res.json();
-        return json;
+        const { status_code, ...rest }: RequestAlias = await res.json();
```

Comment:
only reason there is any logic here at all is just so we can make the responses match... if that gets changed on the backend, then we can drop this.